<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Generator Assignment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_vertices">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Vertices</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_density">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Density of Graph</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_vertices">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
                                                        <!-- Toggle Switch -->
<!--   <widget class="QCheckBox" name="toggleSwitch">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle Switch</string>
    </property>
   </widget>-->
   <!-- Max, Min, Neighbours -->
                                                    <!--Anns Khan copyright section -->
<widget class="QLabel" name="label_anns_khan">
    <property name="geometry">
        <rect>
            <x>50</x>
            <y>550</y>
            <width>211</width>
            <height>16</height>
        </rect>
    </property>
    <property name="text">
        <string>üíªü§ù Anns Khan ¬© </string>
    </property>
</widget>

                                                 <!--max, min, neighbours -->
   <widget class="QLabel" name="label_max_degree">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Node with maximum degree: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_min_degree">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Node with minimum degree: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_find_neighbours">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter node to find its neighbors:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="node_input">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>480</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_find_neighbors">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>480</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Neighbours</string>
    </property>
   </widget>


                                                     <!-- Adjacency Matrix Display -->
     <widget class="QLabel" name="label_adjacency_matrix">
     <property name="geometry">
    <rect>
    <x>400</x>
    <y>150</y>
    <width>121</width>
    <height>16</height>
    </rect>
    </property>
    <property name="text">
    <string>Adjacency Matrix:</string>
    </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_adjacency_matrix">
    <property name="geometry">
    <rect>
    <x>400</x>
    <y>170</y>
    <width>401</width>
    <height>221</height>
    </rect>
    </property>
    <property name="readOnly">
    <bool>true</bool>
   </property>
 </widget>

                                                    <!-- Neighbours Display -->
<!--<widget class="QLabel" name="label_neighbours">
    <property name="geometry">
        <rect>
            <x>50</x>
            <y>450</y>
            <width>221</width>
            <height>16</height>
        </rect>
    </property>
    <property name="text">
        <string>Neighbours of the entered node:</string>
    </property>
</widget>
<widget class="QTextEdit" name="textEdit_neighbours">
    <property name="geometry">
        <rect>
            <x>50</x>
            <y>470</y>
            <width>201</width>
            <height>121</height>
        </rect>
    </property>
    <property name="readOnly">
        <bool>true</bool>
    </property>
</widget>-->
                                                    <!---generate graph button -->
   <widget class="QLineEdit" name="lineEdit_density">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_generate">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Graph</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
